bpf/include/common.h

```diff
@@ -133,11 +133,31 @@ struct {
 
 struct {
     __uint(type, BPF_MAP_TYPE_ARRAY);
-    __uint(max_entries, 1);
+    __uint(max_entries, 3);
```

Comment:
I think we should not use each element to store one option. This may cause the map change frequently.

How about  increase the value size, we use value memory to store a options struct

Reply:
But different options have differenet update frequencies. 
For example the `log_level` will change in use, but the `node IP` and `pod gateway` only need to be passed when the Kmesh-daemon is started.
So I think putting the configurations in one structure will transfer invalid information when updating, resulting in a waste of resources.